web/cobrands/sass/_layout.scss

```diff
@@ -280,8 +331,33 @@ body.mappage.admin {
   .with-notes & {
     width: (($mappage-sidebar-width + $mappage-notes-width) - ($mappage-sidebar-padding * 2));
     // TODO: Should have a bitmap image fallback for old browsers!!
-    background-image: -webkit-linear-gradient(flip(90deg, 270deg), transparent 29em, #E9F2FF 29em);
-    background-image: linear-gradient(flip(90deg, 270deg), transparent 29em, #E9F2FF 29em);
+    background-image: -webkit-linear-gradient(flip(90deg, 270deg), transparent 52%, #E9F2FF 52%);
```

Comment:
I don't think 52% will be the equivalent of the widths given. Can we put the actual same width in, with media queries, so it appears at the right point? Or should we just ditch this and do it another way entirely?